wodles/aws/aws_s3.py

```diff
@@ -227,10 +231,31 @@ def delete_deprecated_tables(self):
             elif 'trail_progress' in table:
                 self.db_connector.execute(self.sql_drop_table.format(table='trail_progress'))
 
-    def get_client(self, access_key, secret_key, profile, iam_role_arn, service_name, bucket, region=None,
+    @staticmethod
+    def default_config():
+        args = {}
+        if not path.exists("".join([path.expanduser('~'), '/.aws/config'])):
```

Comment:
Try to avoid using hard-coded paths with system-specific separators. Use `os.path.join` instead:
```suggestion
        if not path.exists(path.join(path.expanduser('~'), '.aws', 'config')):
```